RealmTasks Apple/RealmTasks macOS/ItemCellView.swift

```diff
@@ -115,7 +115,7 @@ class ItemCellView: NSTableCellView {
         setupUI()
         setupGestures()
 
-        setTrackingAreaWithRect(bounds, options: [.MouseEnteredAndExited, .ActiveInKeyWindow])
+        setTrackingAreaWithRect(rect: bounds, options: [.mouseEnteredAndExited, .activeInKeyWindow])
```

Comment:
Should probably be renamed to `setTrackingArea(with rect:, options:)` and `@discardableResult` should be added to avoid warnings.